### DISCUSSION TITLE
ObservableCollection::Clear with ListView causes crash in .NET6?

### DISCUSSION THREAD

**Discussion by carbon1911** [dotnet/maui#16168] [Q&A]
Hi, I've recently stumbled upon this #9302 issue in my code and it says it's been solved in [7.0.0-rc.2.6866](https://github.com/dotnet/maui/releases/tag/7.0.0-rc.2.6866). My question is whether it's also been solved in .NET6? I'm having troubles finding out whether that fix have been integrated into some .NET6 release. I've used .NET SDK 6.0.412, which I believe is the most recent version, however the app keeps crashing.

This problem is happening when migrating from Xamarin, so maybe it could be worked around by using the CollectionView https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/collectionview/#move-from-listview-to-collectionview.

Thank you for any feedback and feel free to suggest improvements to this question, it is my first asked question ever on github 😇

**Comment by DDHSchmidt** [comment]
The linked PR in the Issue looks like it was merged in to the then-Net7 branch and personally I haven't experienced that stacktrace in .Net 7, so that version seems clear.

That is not to say ObservableCollection+CollectionView is bug-free. We're currently still plagued by Issue #7237 which has remained unsolved for over one year...

**Comment by JRosanowski** [comment]
It seems that bug fixes won't necessarily get back ported. But adding to that .Net Maui end of life policy means versions are EOL sooner than you expect. The .Net Maui 6 EOL is not related to the .Net 6 EOL. .Net Maui 6 went EOL on 23 May...

https://dotnet.microsoft.com/en-us/platform/support/policy/maui
_In order to respond to the shipping cadence of external dependencies (such as Xcode or Android SDK Tools), .NET MAUI is a workload that can be installed independent of installing the .NET SDK. As such, it's important to recognize that support for .NET MAUI works slightly differently from support for .NET, even though .NET MAUI aligns with the .NET release cadence.
**A major version of .NET MAUI receives support for a minimum of 6 months after a successor (the next major release) ships. For example, .NET MAUI 7.0 will receive support for 6 months after .NET MAUI 8.0 ships.**_

**Reply by carbon1911** [reply]
Interesting, so can I use MAUI 7/8 (or whatever MAUI version where #9302 is solved) with .NET6? Thank you :)
